### DISCUSSION TITLE
Use Compromise-dates with French

### DISCUSSION THREAD

**Discussion by edbrdi** [nlp-compromise/nlp_compromise#1005] [Q&A]
Hi, thanks for your work on Compromise! I'm wondering if it's possible to use the plugin compromise-dates with fr-compromise if I translate the folder "words" to French? Maybe it's not that easy and need specific dependencies from compromise not available yet for fr-compromise?

I'll be very happy to contribute to fr-compromise.

**Comment by spencermountain** [comment]
you know what, @arnaudh458 was just asking about this too. We should do it.

I've started a branch called `new-dates` - feel free to check it out. If you run `node scratch.js`, you'll see it does some simple parsing in [./src/03-three/dates](https://github.com/nlp-compromise/fr-compromise/tree/new-dates/src/03-three/dates)

Lots of low-hanging fruit here! The english one creates ISO-formatted strings using [spacetime](https://github.com/spencermountain/spacetime) but we can do whatever we'd like to
cheers

**Comment by spencermountain** [comment]
changed my mind - it needs to use spacetime for things like 'deux mois apres ...', so must be a seperate plugin.
moved some details [over here](https://github.com/nlp-compromise/fr-compromise/issues/23)
cheers
